README.md

```diff
@@ -96,19 +96,23 @@ Also contains pointers where to go next.
     - JDK 17 or newer
     - [Maven](https://maven.apache.org/download.cgi)
     - [Terraform](https://developer.hashicorp.com/terraform/downloads)
-    - [Python 3.8 or newer](https://www.python.org/downloads/)
+    - Python 3.8
```

Comment:
will it not work with newer python?

Reply:
It didn't work for me, due to some changes in some dependencies. Therefore, I wanted to fix python version and dependency versions. Newer python versions can't even install some the old dependency versions in the requirements.txt.

Reply:
I use Python 3.12.3 and it works, but I configured it some time ago and I did not use `pip install -U -r requirements.txt`